docs/maui/markup/extensions/element-extensions.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Element Extensions - .NET MAUI Community Toolkit
```

Comment:
```suggestion
title: Element extensions - .NET MAUI Community Toolkit
```

I've seen @bijington make this change to all the titles (also below on line 8). Probably @davidbritch might've mentioned that this is a guideline from MS?